core/src/trezor/ui/__init__.py

```diff
@@ -306,6 +310,9 @@ class Layout(Component):
     BACKLIGHT_LEVEL = style.BACKLIGHT_NORMAL
     RENDER_SLEEP = loop.sleep(_RENDER_DELAY_MS)  # type: loop.Syscall
 
+    def __init__(self) -> None:
+        super().__init__()
+
```

Comment:
Isn't this implicit?

Reply:
IIUC it is, will remove

Reply:
Fixed in 63e2372c1831791ada10e52f362b6e8f764bc3fa.